draft-ietf-masque-h3-datagram.md

```diff
@@ -193,8 +193,8 @@ allow communicating prioritization preferences.
 
 ### The SETTINGS_H3_DATAGRAM HTTP/3 Setting {#setting}
 
-Implementations of HTTP/3 that are willing to receive HTTP Datagrams can
```

Comment:
Generally we're using "HTTP Datagrams" to mean things that work even with capsules, and "HTTP/3 Datagrams" to mean the things that go in QUIC datagrams. So this seemed incorrect.